docs/reference/transform/apis/start-transform.asciidoc

```diff
@@ -27,16 +27,20 @@ Requires the following privileges:
 [[start-transform-desc]]
 == {api-description-title}
 
-When you start a {transform}, it creates the destination index if it
-does not already exist. The `number_of_shards` is set to `1` and the
+When you start a {transform}, it creates the destination index if it does not 
+already exist. The `number_of_shards` is set to `1` and the
 `auto_expand_replicas` is set to `0-1`.
 
-The {transform} deduces the mapping definitions from the source indices. For
-scripted fields, it uses <<dynamic-mapping,dynamic mappings>>. If a field in the
-destination index is created by `scripted_metric` or `bucket_script`
-aggregations, the {transform} uses dynamic mappings unless a template exists or
-the destination index already exists. Mapping definitions in the destination
-index take precedence over dynamic mappings and templates.
+If it is a pivot {transform}, it deduces the mapping definitions for the
+destination index from the source indices and the transform aggregations. If
+fields in the destination index are derived from scripts (as in the case of
+`scripted_metric` or `bucket_script` aggregations), the {transform} uses
+<<dynamic-mapping,dynamic mappings>> unless an index template exists.
+
+If it is a latest {transform}, it does not deduce mapping definitions; it uses
+dynamic mappings.
+
+TIP: To use explicit mappings, create the destination index before you start the {transform}.
```

Comment:
I think the tip should also mention the option to leverage index template.
If the destination index name matches the template pattern, the destination index is created by the transform with the template-supplied mappings.

Reply:
This is only true for the latest transform and for the pivot transform where it would have otherwise used dynamic mappings, though, right?

Reply:
> This is only true for the latest transform and for the pivot transform where it would have otherwise used dynamic mappings, though, right? 

Yes, that's my understanding too.

Reply:
Thanks! I added that info in 2a48718023b92cfe8d553445347760f0f58d9f65